cmd/jira-lifecycle-plugin/server_test.go

```diff
@@ -4598,6 +4638,8 @@ func checkComments(client *fakegithub.FakeClient, name, expectedComment string,
 
 	if expectedComment != "" && len(client.IssueCommentsAdded) == 1 {
 		if expectedComment != client.IssueCommentsAdded[0] {
+			fmt.Printf("Expected: \n====\n%v\n====\n", expectedComment)
+			fmt.Printf("CommentAdded: \n====\n%v\n====\n", client.IssueCommentsAdded[0])
```

Comment:
Same with these printf. The `t.Errorf` below should print the diff, so these are unnecessary for the tests.